testem.json

```diff
@@ -3,8 +3,15 @@
   "cwd": ".dist-test",
   "test_page": "index.html?hidepassed",
   "disable_watching": true,
+  "launchers": {
+    "NodeQUnit": {
+      "command": "node scripts/util/test-in-node.js",
```

Comment:
so the thing is, it can not be run directly. It depends on execution of `scripts/test` which uses `testem`. That's why i moved it into util. Wasn't sure where exactly it fits